lib/logged_cheetah.rb

```diff
@@ -16,12 +16,30 @@
 # you may find current contact information at www.suse.com
 
 class LoggedCheetah
-  def self.run(*args)
-    command = args.select{|e| e.is_a?(String)}.join(" ")
-    Machinery.logger.info("Running '#{command}'")
+  class << self
+    def run(*args)
+      run_overloaded(*args, {})
+    end
+
+    def run_with_c(*args)
+      run_overloaded(*args, with_c_locale: true)
+    end
+
+    private
+
+    def run_overloaded(*args, options)
+      command = args.select{|e| e.is_a?(String)}.join(" ")
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.